### DISCUSSION TITLE
Invalid attachment error

### DISCUSSION THREAD

**Discussion by kartal-huseyin** [test-results-reporter/reporter#126] [Ideas]
![image](https://user-images.githubusercontent.com/98432117/228482891-cccf53ab-033b-44de-8653-f91e3d79ccce.png)

"publish": "failure-details" should not be limited

While there is no problem publishing with "test-summary" and "test-summary-slim", there is an error at "failure-details" while having more than 30 errors at the tests. 

![image](https://user-images.githubusercontent.com/98432117/228482378-2dfd4d72-67cc-4bc3-890e-adf355a5e83d.png)

**Comment by ASaiAnudeep** [comment]
I think we hit the max number of attachments allowed per message. Marking it as a bug.

**Reply by kartal-huseyin** [reply]
Hi Sai, do you think we could solve this one?

**Reply by ASaiAnudeep** [reply]
Is it happening with MS Teams or Slack?

Ideally this [release](https://github.com/test-results-reporter/testbeats/releases/tag/v1.1.1) should have fixed it.

**Comment by kartal-huseyin** [comment]
Nope, it is not solved. I was just trying on Slack, Now I've updated the
latest and tried with MS Teams too. It is still happening. It's fine with
slim or summary.

**Reply by ASaiAnudeep** [reply]
Can you please share the config file and the results file for us to investigate it further.

**Comment by kartal-huseyin** [comment]
Hi Sai, sorry for the late response.

The result file and config are attached.

Teams have worked with more than 30 failures.

When I changed slack config to this ""publish": "test-summary-slim"," it
works.
When I changed slack config to this ""publish": "failure-details"," it
fails when the failed results is over 30.

**Reply by ASaiAnudeep** [reply]
Hello @kartal-huseyin , there is a limit on home many attachments each platform would support. It is best to use the `max_suites` config property to limit the size of the message. And also I'm not able to find the config and result file.
